lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -29,25 +29,25 @@
 options:
   source_region:
     description:
-      - The source region the AMI should be copied from.
+      - the source region that AMI should be copied from
     required: true
   source_image_id:
     description:
-      - The ID of the AMI in source region that should be copied.
+      - the id of the image in source region that should be copied
     required: true
   name:
     description:
-      - The name of the new AMI to copy.
-    required: false
+      - The name of the new image to copy
+    required: true
```

Comment:
The name field is a requirement in boto3/AWS API. An alternative option may be to set the default to an acceptable value such as 'default' to avoid breaking changes. I'll add a default value and avoid making it required for now. Let me know how you would like to handle it in the long term.

`AMI names must be between 3 and 128 characters long, and may contain letters, numbers, '(', ')', '.', '-', '/' and '_'"`